superset/reports/notifications/email.py

```diff
@@ -89,11 +94,15 @@ def _get_content(self) -> EmailContent:
             html_table = ""
 
         call_to_action = __("Explore in Superset")
-        img_tag = (
-            f'<img width="1000px" src="cid:{msgid}">'
-            if self._content.screenshot
-            else ""
-        )
+        img_tag = ""
+        for msgid, _ in images.items():
```

Comment:
Nit:

```suggestion
        for msgid in images.keys():
```

(Or just `images` instead of `images.keys()`, though I think explicit is better.)